src/github/middleware.ts

```diff
@@ -62,38 +72,60 @@ export default (
 ) => {
 	return withSentry(async (context: CustomContext) => {
 		enhanceOctokit(context.github);
+		context.webhookReceived = getCurrentTime();
 
 		let webhookEvent = context.name;
 		if (context.payload.action) {
 			webhookEvent = `${webhookEvent}.${context.payload.action}`;
 		}
 
+		const webhookReceived = context.webhookReceived || undefined
```

Comment:
We don't really need `|| undefined` here hence we assigned a value on the line 75 (And if the field webhookRecevied wasn't there it would be undefined anyway)

Maybe it is better to do 
``` 
 const webhookReceived = getCurrentTime()
 context.webhookReceived = webhookRecevied
```